nix/build/install_topfind_196.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/findlib/Makefile b/src/findlib/Makefile
```

Comment:
Shall we open a PR to the `opam-nix` upstream to include this patch?

Reply:
I'm not sure TBH, look at this [commit](https://github.com/tweag/opam-nix/commit/4ea5dac926f1d8e031e56001449fd98f8c7fca0b)

Reply:
I don't really get why it is unneeded, because I should fail the same way as other versions of `findlib`